src/index.ts

```diff
@@ -25,23 +26,23 @@ export const run = async function (options?: RunOptions): Promise<BuildContext |
     esbuildOptions = esbuildOptionsFn(args, esbuildOptions);
   }
 
-  const errorHandler = (err: any): void => {
-    console.log(err);
-    process.exit(1);
-  };
+  touchManifest(root);
 
   if (args.watch) {
-    touchManifest(root);
-
     const ctx = await esbuild.context(esbuildOptions);
     await ctx.watch().catch(errorHandler);
 
     return ctx;
```

Comment:
Perhaps a silly question, why are multiple builds only run for the production build, but not the watcher build?

Reply:
@KonnorRogers I think this is Luca's next step. We definitely need similar behavior for watch mode.